kratos/python_interface/python_registry.py

```diff
@@ -39,6 +39,7 @@ class PythonRegistry(object):
 
     #TODO: Change this by a frozenmap() once this is available
     CppGetFunctionNamesMap = {
+        "Modelers": "GetModeler",
         "Operations": "GetOperation",
         "Processes": "GetProcess"
     }
```

Comment:
probably a `dataclass` would be better her. That one you can make `frozen`!

~~~py
from dataclasses import dataclass

@dataclass(frozen=True)
class CppFunctionNames:
    Modelers: str = "GetModeler"
    Operations: str = "GetOperation"
    Processes: str = "GetProcesse"
~~~

this is even type safe, maybe works better for you

Reply:
Nice. I'll add it in a follow up PR (just in case).